osu.Server.Queues.ScoreStatisticsProcessor/ScoreStatisticsQueueProcessor.cs

```diff
@@ -211,6 +220,16 @@ private IEnumerable<IProcessor> enumerateValidProcessors(SoloScoreInfo score)
             return result;
         }
 
+        private static void updateLegacyScorePerformance(SoloScoreInfo score, MySqlConnection db, MySqlTransaction transaction)
+        {
+            string highScoresTable = LegacyDatabaseHelper.GetRulesetSpecifics(score.RulesetID).HighScoreTable;
+            db.Execute($"UPDATE {highScoresTable} SET pp = @Pp WHERE score_id = @LegacyScoreId", new
```

Comment:
For scores imported from `osu_scores` (the non-`high` table), `LegacyScoreId` will be 0.

I'm not sure how to handle this - do these scores need to be updated? @peppy

Reply:
Yeah, that was the clarification I needed. I think a `score.IsLegacyScore && score.LegacyScoreId > 0` should suffice here